_rules/SC3-1-1-html.md

```diff
@@ -2,6 +2,7 @@
 rule_id: SC3-1-1-html
 name: Primary language of page
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```

Comment:
Step 1: *Do nothing. (This case is covered by [SC3-1-1-xml-lang](SC3-1-1-xml-lang.html).)*
Suggestion: *Execute rule SC3-1-1-xml-lang*